content/docs/base-components/ReactiveComponent.md

```diff
@@ -0,0 +1,83 @@
+---
+id: reactivecomponent
+title: "Reactive Component"
+layout: docs
+sectionid: reactivecomponent
+permalink: base-components/reactivecomponent.html
+prev: selectedfilters.html
+prevTitle: "SelectedFilters"
+next: /list-components/singlelist.html
+nextTitle: "List Components: SingleList"
+redirect_from:
+    - 'basic-components/reactivecomponent.html'
+    - 'base-components/reactivecomponent'
+    - 'reactivecomponent'
+---
+
+
+With `ReactiveComponent`, you can convert any React Component into a Reactivesearch component i.e. your react component will be able to talk to other Reactivesearch components and connect with your elasticsearch cluster seamlessly. `Note` This page only cover `ReactiveComponent` with `customQuery`. Read more [here](/advanced/reactivecomponentindepth.html).
```

Comment:
ReactiveComponent lets you connect any React UI component with an ElasticSearch query or an aggregation seamlessly. It can be used as a standalone component to integrate ElasticSearch queries into your frontend UI declaratively. It can also be used in [conjunction with other ReactiveSearch components](link-to-advanced-docs).